### DISCUSSION TITLE
站点设置完成后【我的数据】内无任何信息 故障求助

### DISCUSSION THREAD

**Discussion by laputa173** [ronggang/PT-Plugin-Plus#1462] [Q&A] [answered]
![image](https://github.com/pt-plugins/PT-Plugin-Plus/assets/85661778/5945b421-3af3-4a3a-a024-5828ce5932ac)

**Comment by laputa173** [comment]
补充【我的数据】内截图供参考，仍然显示请前往站点设置。win10专业版+edge
![image](https://github.com/pt-plugins/PT-Plugin-Plus/assets/85661778/b2cec956-e809-465c-886c-9a5c464a8be0)

**Comment by ted423** [comment]
非标issue

**Comment by ted423** [comment]
@laputa173 issue是说明程序问题的地方，问问题去discussion。

**Comment by ted423** [comment]
@laputa173 另外这种情况可以f5刷新几次看看

**Comment by ted423** [comment]
@ted423 issue里选bug是有模板的，譬如说明扩展版本，浏览器环境等很多东西，这样才好排查

**Comment by ted423** [comment]
才发现我可以转成discussion

**Comment by laputa173** [comment, marked as answer]
谢谢。家里电脑上的问题已解决，具体过程我已描述在  [https://github.com/pt-plugins/PT-Plugin-Plus/discussions/1461]
但刚刚远程公司电脑，发现EDGE内问题依旧，但至少时间轴功能里有显示了。

**Reply by mitlearn** [reply]
过滤列取消显示等级或使用非商店版
